samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -176,4 +182,10 @@ static void RegisterViewModels(in IServiceCollection services)
 		services.AddTransient<PopupPositionViewModel>();
 		services.AddTransient<XamlBindingPopupViewModel>();
 	}
-}
+
+	static void RegisterEssentials(in IServiceCollection services)
+	{
+		services.AddSingleton<IDeviceInfo, DeviceInfoImplementation>();
```

Comment:
Does static DeviceInfo use the injected class or it create a new instance?
`
DeviceInfo.Current == services.GetService<IDeviceInfo>???
`